src/ripple/app/tx/impl/SetSignerList.cpp

```diff
@@ -281,23 +283,23 @@ SetSignerList::removeSignersFromLedger (Keylet const& accountKeylet,
     if (!signers)
         return tesSUCCESS;
 
-    STArray const& actualList = signers->getFieldArray (sfSignerEntries);
+    STArray const& actualList = signers-> getFieldArray (sfSignerEntries);
```

Comment:
nit: extra space

Reply:
Not sure how I did that! Just fixed it, though.